rgw/admin/bucket_quota_test.go

```diff
@@ -0,0 +1,27 @@
+package admin
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func (suite *RadosGWTestSuite) TestBucketQuota() {
+	suite.SetupConnection()
+	co, err := New(suite.endpoint, suite.accessKey, suite.secretKey, newDebugHTTPClient(http.DefaultClient))
+	assert.NoError(suite.T(), err)
+
+	suite.T().Run("set bucket quota but no user is specified", func(t *testing.T) {
+		err := co.SetIndividualBucketQuota(context.Background(), IndividualBucketQuotaSpec{})
+		assert.Error(suite.T(), err)
+		assert.EqualError(suite.T(), err, errMissingUserID.Error())
+	})
+
+	suite.T().Run("set bucket quota but no bucket is specified", func(t *testing.T) {
+		err := co.SetIndividualBucketQuota(context.Background(), IndividualBucketQuotaSpec{UID: "admin"})
+		assert.Error(suite.T(), err)
+		assert.EqualError(suite.T(), err, errMissingUserBucket.Error())
+	})
```

Comment:
Where is the successful test?

Reply:
@leseb thank you for your review and fixes. Added commit with positive case test and fix of idempotentness of keys API tests